include/internal/time.h

```diff
@@ -81,4 +87,45 @@ OSSL_TIME ossl_time_subtract(OSSL_TIME a, OSSL_TIME b)
     return err ? 0 : r;
 }
 
+/* Returns |a - b|. */
+static ossl_unused ossl_inline
+OSSL_TIME ossl_time_difference(OSSL_TIME a, OSSL_TIME b)
```

Comment:
I suggest `ossl_time_abs_difference()` for the name.

Plus these new functions & macros need to be documented in doc/internal/man3/OSSL_TIME.pod.